ls-ui-react-app/src/pages/UploadCircuitPage.tsx

```diff
@@ -87,21 +97,23 @@ const UploadACircuit = ( {appState, setAppState} : AppStateProps ) => {
                     Transform</label>
             </div>
         </form>
-    </p>
+    </div>
 }
 
 // TODO implement default circuits by pulling them from assets
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const SelectSampleCircuit = ({setAppState} : AppStateProps) => <p>
```

Comment:
This is disabled right now ... might be worth adding back some day, but there likely are higher priority issues first :thinking: